src/main/java/org/jabref/logic/importer/util/MathMLParser.java

```diff
@@ -0,0 +1,69 @@
+package org.jabref.logic.importer.util;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Objects;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+import javax.xml.transform.Result;
+import javax.xml.transform.Source;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
```

Comment:
Shouldn't this be jakarta.xml instead?

Reply:
No, this is the normal jdk stuff since java 1.6. We aren't using jaxb and therefore don't have to deal with jakarta.

Reply:
Ok.